### PR TITLE
safe-log errorIds

### PR DESCRIPTION
**Pull request by pnepywoda**
This change is [Reviewable](https://reviewable.io/reviews/palantir/http-remoting/456)

### CODE REVIEW

**Comment by pnepywoda** [comment]
@uschi2000

**Comment by uschi2000** [comment]
@lcuthriell does this look ok?

File: jersey-servers/build.gradle [hunk 1]
```diff
@@ -9,6 +9,7 @@ dependencies {
     compile "com.fasterxml.jackson.module:jackson-module-afterburner"
     compile "com.jcraft:jzlib"
     compile "org.glassfish.jersey.core:jersey-server"
+    compile "com.palantir.safe-logging:safe-logging"
```
Comment (gracew) [inline jersey-servers/build.gradle]: alpha

**Review by gracew — APPROVED** [review]
lgtm

**Comment by gracew** [comment]
we should also audit other log statements - https://github.com/palantir/http-remoting/issues/457

File: versions.props [hunk 2]
```diff
@@ -7,6 +7,7 @@ com.google.guava:guava = 18.0
 com.jcraft:jzlib = 1.1.3
 com.netflix.feign:feign-*= 8.17.0
 com.palantir.tokens:auth-tokens = 1.0.0
+com.palantir.safe-logging:safe-logging = 0.1.3
```
Comment (schlosna) [inline versions.props]: nit: `%!sort`

**Pull request merged**